Infrastructure/Music_Portal.Infrastructure.Data/ArtistRepository.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Domain.Core;
+using Domain.Interfaces;
+
+namespace Infrastructure.Data
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
`namespace Music_Portal.Infrastructure.Data`